spec/features/contestants/index_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+RSpec.describe "the contestants index" do
+  before(:each) do
+    @recycled_material_challenge = Challenge.create(theme: "Recycled Material", project_budget: 1000)
+    @furniture_challenge = Challenge.create(theme: "Apartment Furnishings", project_budget: 1000)
+
+    @news_chic = @recycled_material_challenge.projects.create(name: "News Chic", material: "Newspaper")
+    @boardfit = @recycled_material_challenge.projects.create(name: "Boardfit", material: "Cardboard Boxes")
+
+    @upholstery_tux = @furniture_challenge.projects.create(name: "Upholstery Tuxedo", material: "Couch")
+    @lit_fit = @furniture_challenge.projects.create(name: "Litfit", material: "Lamp")
+
+    @jay = Contestant.create(name: "Jay McCarroll", age: 40, hometown: "LA", years_of_experience: 13)
+    @gretchen = Contestant.create(name: "Gretchen Jones", age: 36, hometown: "NYC", years_of_experience: 12)
+    @kentaro = Contestant.create(name: "Kentaro Kameyama", age: 30, hometown: "Boston", years_of_experience: 8)
+
+
+    ContestantProject.create(contestant_id: @jay.id, project_id: @news_chic.id)
+    ContestantProject.create(contestant_id: @gretchen.id, project_id: @news_chic.id)
+    ContestantProject.create(contestant_id: @gretchen.id, project_id: @upholstery_tux.id)
+    ContestantProject.create(contestant_id: @kentaro.id, project_id: @upholstery_tux.id)
+    ContestantProject.create(contestant_id: @kentaro.id, project_id: @boardfit.id)
+  end
+
+  it "list all contestants and their projects" do
+    visit "/contestants"
+
+    expect(page).to have_no_content(@lit_fit.name)
+    expect(page).to have_content(@jay.name)
+    expect(page).to have_content(@gretchen.name)
+    expect(page).to have_content(@kentaro.name)
+    expect(page).to have_content("Projects: News Chic")
+    expect(page).to have_content("Projects: News Chic, Upholstery Tuxedo")
+    expect(page).to have_content("Projects: Boardfit, Upholstery Tuxedo")
+  end
```

Comment:
Lines `33` and `34` have the potential to produce a false-positive because they both list `"News Chic"` first. I think to make this test a little better, it would have been a good idea to introduce some `within` blocks.

```ruby
expect(page).to have_no_content(@lit_fit.name)

within "#contestant-#{@jay.id}" do
  expect(page).to have_content(@jay.name)
  expect(page).to have_content("Projects: "New Chic")
end

within "#contestant-#{@gretchen.id}" do
  expect(page).to have_content(@gretchen.name)
  expect(page).to have_content("Projects: "New Chic, Upholstery Tuxedo")
end

within "#contestant-#{@kentaro.id}" do
  expect(page).to have_content(@kentaro.name)
  expect(page).to have_content("Projects: Boardfit, Upholstery Tuxedo")
end
```

Great job using `"Projects:"` as anchor text to target a specific wording on your page.